README.md

```diff
@@ -30,6 +30,40 @@ The benchmarks that are here by default are made to test the performance of Phar
 2. [Optional] Add a `.sh` file in the `/vms` folder with the instructions to download your vm. If not you can use one of the already present vms.
 3. Execute `runBenchs.sh name_of_your_benchmark` with the name.s of your benchmark.s
 
+# Create your own configuration
+
+You can create your own benchmark configuration by creating a `.sh` file using the following template:
+
+```bash
+#!/bin/bash
+
+IMAGES="<the images you want your benchmark to run in>"
+VMs="<the VMs you want your benchmark to run with>"
+PHARO_CMD="<the command you want the VM to execute>"
+
```

Comment:
maybe you can add the `ITERATIONS`  and the `VM_PARAMETERS` because one can configure that also

Reply:
I have a question about this. If I change for example the number of iterations for a bench, it affects the next benches or only that?

Reply:
I t

> I have a question about this. If I change for example the number of iterations for a bench, it affects the next benches or only that?

I think it will affect all subsequent benches. Interesting question :)
This means that eventually, for safely, we would like that each bench is run in a clean environment?

Reply:
and ok guille I changed to accepted

Reply:
Ee need tests? 🤔